cmake/modules/FindPython.cmake

```diff
@@ -0,0 +1,224 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
+# file Copyright.txt or https://cmake.org/licensing for details.
+
+#[=======================================================================[.rst:
+FindPython
```

Comment:
if we're going to maintain these files in-tree, it would help to add a comment saying which cmake version they were taken from. that way we'll know when it's safe to remove them after updates to cmake_minimum_required

Reply:
good point! will do so in next revision.

Reply:
updated and repushed.